dist/.magento.env.yaml

```diff
@@ -336,6 +336,21 @@
 #                    initStatements: SET NAMES utf8                                                                   #
 #                    active: 1                                                                                        #
 #######################################################################################################################
+# RESOURCE_CONFIGURATION - use to configure mapping of resource name to a connection.                                 #
+#                         By default, the deployment process overwrites all settings in the env.php file.             #
```

Comment:
By default, the deployment process overwrites all settings in the `env.php` file; however, you can choose to merge one or more values for a service configuration without overwriting all of the values.

Set the `_merge` option to one of the following:

`true`—Merge the configured service values with the environment variable values.
`false`—Overwrite the configured service values with the environment variable values.